packages/hardhat/hardhat.config.ts

```diff
@@ -51,43 +51,43 @@ const config: HardhatUserConfig = {
       },
     },
     mainnet: {
-      url: `https://eth-mainnet.alchemyapi.io/v2/${providerApiKey}`,
+      url: `https://cloudflare-eth.com`,
```

Comment:
nit question: How to decide which one to use here? As I understand you took one of the stable ones, right?

Reply:
Yess lol I just copied what viem/chains uses for their public rpc URLs (Assuming lol they did some research on it). 

Also was thiking in future, instead of hardcoding this URL's we could bring viem/chains use the url from their so that when viem/chain updates we get updated URL's too. But yeah it has it pro's and con's

Reply:
Make sense, thanks! Agree, it's better to use import that URLs later then